modules/gentemplates/openapi-generator/src/main/java/com/reprezen/genflow/openapi/generator/OagCodegenGenTemplateBase.java

```diff
@@ -64,6 +64,32 @@ public void configure() throws GenerationException {
 				"Each property should be a JSON object with a name/value pair for each property.",
 				"Example: for '-Dmodels -Dapis=User,Pets' use the following:", "value:", "  models: ''",
 				"  apis: Users,Pets"));
+		define(parameter().named(LANGUAGE_SPECIFIC_PRIMITIVES).optional().withDescription(
+				"Specifies additional language specific primitive types.",
+				"You can have multiple occurrences of this option.", //
+				"Example: -Pet", //
+				"         -User"));
+		define(parameter().named(TYPE_MAPPINGS).optional().withDescription(
+				"Sets mappings between Swagger spec. types and generated code types.", //
+				"You can have multiple occurrences of this option.", //
+				"Example: array:List", //
+				"         map:Map"));
```

Comment:
Same comments as above apply here, and to the remaining parameters, as appropriate.

1. "Type names here should be unqualified. The fully-qualified name should be in IMPORT_MAPPINGS."
2. Use the above language to explain how multiple values are array elements, and change the format of the example to make it more realistic.

Also:  `array:List` is not legal YAML; it needs a space after the colon.  All of these examples need that correction.